internal/processor/filterconfig/config.go

```diff
@@ -1,53 +1,56 @@
-// Copyright The OpenTelemetry Authors
+// Copyright  The OpenTelemetry Authors
```

Comment:
Is this an accidental change or needed for something?

Reply:
@keitwb ^^^

Reply:
```suggestion
// Copyright The OpenTelemetry Authors
```